utilities.go

```diff
@@ -77,6 +78,20 @@ func OuterHtml(s *Selection) (string, error) {
 	return buf.String(), nil
 }
 
+// Render renders first item in the selection (like in OuterHtml - that is,
+// the HTML including the first element's tag and attributes) to the given writer
+// by calling `html.Render`. In case of empty `s`, empty byte slice will be written.
+//
+// In order to render entire document, pass a `doc.Selection` as `s`.
```

Comment:
Small nit: Go's comments do not use markdown syntax, the community convention is to just use standard text (i.e. like the standard library's documentation). Let's remove the backticks.

Reply:
Yep. I wasn't sure about this, fixed.